src/main/resources/i18n/messages_de_DE.properties

```diff
@@ -43,37 +43,37 @@ leaderboard.loading=Lade Rangliste\u2026
 ranked1v1.players=Spieler
 ranked1v1.play=Spiel
 ranked1v1.cancel=Abbrechen
-ranked1v1.choseFactions.title=W�hle eine Fraktion
+ranked1v1.choseFactions.title=W�hle eine Fraktion
```

Comment:
Oh damn. How about I fix this by replacing every non-iso-8859-1 character with its unicode escape sequence like described here: https://docs.oracle.com/javase/8/docs/api/java/util/Properties.html? 
Should be more fail-save and "correct" then, isn't it?